### ISSUE TITLE
Clear previous output when waiting for server

### ISSUE THREAD

**Issue by hauntsaninja** [erictraut/pyright-playground#12]
I sometimes get caught out by the playground saying "No problems detected" when it's waiting for the server

**Comment by erictraut** [comment]
I agree this is confusing, but I'm not sure that clearing the old output is the right answer.

The confusion wouldn't be an issue if the playground responded faster (within a second). I spent a bunch of time this past week working on speeding up the playground. This required changes in the playground frontend, backend, and pyright itself. I've already deployed the playground changes, but the changes to pyright won't be visible until the next version is published (later today). Let's see if the speed improvements mitigate the issue. If not, I'll think about other ways to address this using UI affordances.
